Sources/BraintreeCore/BTClientMetadata.swift

```diff
@@ -11,61 +13,39 @@ import Foundation
 /// the sessionID should remain constant. To achieve this, users of this class
 /// should use `mutableCopy` to create a new copy based on the existing session
```

Comment:
We should get rid of these docs that mention making a `mutableCopy` since we no longer implement `NSMutableCopying` here.

Reply:
Good call, done: https://github.com/braintree/braintree_ios/pull/1052/commits/5fdb7ac2347176bff1f2adc82917e548f49a1aa0